City-of-Pori/pori-kada
# PORI-X Minor fix for topics placeholder

To test:

1. drush cc all
2. Go to page that has topics without images. Resize browser to 320px width and make sure that the "ban" icon is still inside the container like in the disclosed image.

<img width="315" alt="screen shot 2017-11-07 at 12 36 11" src="https://user-images.githubusercontent.com/2276077/32489328-468e6f42-c3b8-11e7-8271-ceb65a91fc11.png">
